src/config/dynamic-theme-fixes.config

```diff
@@ -1065,6 +1065,19 @@ INVERT
 
 ================================
 
+aprs.fi
+*.aprs.fi
+
+INVERT
+.gm-style
+.gm-style-iw-a
```

Comment:
Hmm I don't seem to find this selector.

Reply:
This one is displayed when any node is clicked on the map:
![image](https://user-images.githubusercontent.com/17483811/124500565-48d04480-ddc0-11eb-8c94-0fdda697e4c8.png)